src/interaction/include/exaDEM/interaction/classifier.hpp

```diff
@@ -24,6 +24,24 @@ namespace exaDEM
     ONIKA_HOST_DEVICE_FUNC inline Vec3d& friction(const uint64_t idx) { return m_data[idx].friction; }
   };
 
+
```

Comment:
add deflection (overlap) d_n to the list

Reply:
done